app/serializers/subject_serializer.rb

```diff
@@ -9,6 +9,11 @@ class SubjectSerializer
 
   can_include :project, :collections, :subject_sets
 
+  def self.page(params = {}, scope = nil, context = {})
+    scope = scope.preload(:locations, :project, :collections, :subject_sets)
```

Comment:
Do we need this context for the subject selector results? Should we use a serializer that doesn't link these associations e.g. cards serializer for projects..?